packages/flutter/lib/src/widgets/_web_image_web.dart

```diff
@@ -238,4 +241,8 @@ class _CachedImageInfo {
       _error = e;
     }
   }
+
+  void dispose() {
```

Comment:
As it is now, `dispose()` is called by `_remove()`, which itself is called by `_clear()`.

But `_clear()` itself it not used, so in effect `dispose()` is never called?

Reply:
Ack. I am currently discussing a better way to do this caching for now. One idea is to expose a `WebImageResource` on the `WebImage` which is either backed by an `ImageProvider` or an `<img>` element. Developers can cache this `WebImageResource` themselves and can create a `WebImage` directly using the `WebImageResource`. I think I will also dispose of the state after the image is done loading, since no more state changes will occur after that point anyways.

Reply:
I dispose of the `ValueNotifier` now after the image has finished decoding. However, I still need to refactor how we will do the caching for the initial implementation.